sdk/cosmosdb/cosmos/src/queryExecutionContext/pipelinedQueryExecutionContext.ts

```diff
@@ -98,15 +148,29 @@ export class PipelinedQueryExecutionContext implements ExecutionContext {
         );
       }
     } else {
+      // Create shared continuation token manager for streaming execution contexts
+      const sharedContinuationTokenManager = new ContinuationTokenManager(
+        this.collectionLink,
+        this.options.continuationToken,
+        isOrderByQuery,
+      );
+      
+      // Pass shared continuation token manager via options
+      const optionsWithSharedManager = {
+        ...this.options,
+        continuationTokenManager: sharedContinuationTokenManager
```

Comment:
create an extended feed option for passing continuation token manager

Reply:
Or pass continuationTokenManager directly to the EndpointComponents.